go.mod

```diff
@@ -5,7 +5,7 @@ require (
 	github.com/GoogleContainerTools/kaniko v1.9.2-0.20220928141902-4d077e2a4084
 	github.com/apex/log v1.9.0
 	github.com/awslabs/amazon-ecr-credential-helper/ecr-login v0.0.0-20230110223219-40efa3093a22
-	github.com/buildpacks/imgutil v0.0.0-20230120191822-4d50b9a7e215
+	github.com/buildpacks/imgutil v0.0.0-20230217231607-8def05088e09
```

Comment:
This should be bumped to 4cf98dd677d28557679e4f08941e891f640410cd

Reply:
Let me do that!